### ISSUE TITLE
Prepare application for round 2 interviews

### ISSUE THREAD

**Issue by jduss4** [cloud-gov/cg-ui#306]
Round 1 of user research (#267 ) involved only TTS employees due to access restrictions for the cloud dev environment.

We need to answer some questions around our round 2 user research before we can prepare adequately.

Questions:
- will this be Figma walkthrough / screenshots only or involve a deployed version of the application?
- do we want to test with non-TTS users?

If we will be using the application with non-TTS users, we need to find a suitable deployment environment to test with and set up an org / space / users that can be altered / destroyed / etc without affecting users' production organizations.

### Acceptance Criteria
- [ ] user research focus for round 2 has been selected
- [ ] if testing with the application, steps to prepare environment for testers have been identified and new issue created
